### DISCUSSION TITLE
Welcome to the new home for Kurento discussions!

### DISCUSSION THREAD

**Discussion by j1elo** [Kurento/kurento#9] [Announcements]
## 👋 Welcome!

We’re moving the Kurento discussion forum, from its previous home at the [Kurento group](https://groups.google.com/g/kurento) to GitHub Discussions, a place to connect with other users of the Kurento Media Server. We hope that you:

* Ask questions you’re wondering about
* Share ideas
* Engage with other community members
* Welcome others and are open-minded. Remember that this is a community we build together 💪

**Comment by lvhonglong** [comment]
For Kurento 7.0, I cannot import the custom module that I successfully compiled into it. How should I solve it？
